tools/cpp/windows_cc_configure.bzl

```diff
@@ -323,6 +323,9 @@ def _get_vc_env_vars(repository_ctx, vc_path, msvc_vars_x64, target_arch):
         lib = msvc_vars_x64["%{msvc_env_lib_x64}"]
         full_version = _get_vc_full_version(repository_ctx, vc_path)
         tools_path = "%s\\Tools\\MSVC\\%s\\bin\\HostX64\\%s" % (vc_path, full_version, target_arch)
+        # For native windows on arm64 builds host toolchain runs in an emulated x86 environment
```

Comment:
Wow, do you mean there is no C++ build tools that runs natively on Windows arm64 platform? 

Also, can you make it a parameter instead of using "replace", I think that's safer.
```
tools_path = "%s\\Tools\\MSVC\\%s\\bin\\%s\\%s" % (vc_path, full_version, host_arch, target_arch)
```

Reply:
Yes, they all run in emulated mode now.

For Windows 11, x64 toolchain is available so I want to keep the logic of checking for x64 toolchain first and then fallback to x86 if x64 toolchain is not available.